client/src/components/graph/graph.tsx

```diff
@@ -929,24 +963,82 @@ class Graph extends React.Component<GraphProps, GraphState> {
     return createColorQuery(colorMode, colorAccessor, schema, genesets);
   }
 
+  updateOpenSeadragon() {
+    const {
+      viewport: { width, height },
+    } = this.state;
+
+    const {
+      config: { s3URI },
+    } = this.props;
+
+    if (
+      this.openseadragon ||
+      !shouldShowOpenseadragon(this.props, this.state) ||
+      !width ||
+      !height
+    ) {
+      return;
+    }
+
+    this.openseadragon = Openseadragon({
+      // (thuang): This id will need to be unique for multiple graphs
+      id: "openseadragon",
```

Comment:
CC: @seve when we have side by side mode, I think we'll need to update this id to be unique!